python/functional-programming/functional-programming/why-functional-programming.md

```diff
@@ -21,14 +21,14 @@ links:
 
 ---
 
-# Why Functional Programming?
+# Why Functional Programming? (Part 1)
```

Comment:
if we're gonna change the headlines for part 2, 3, and 4 there is no need to include Part 1 anymore in this headline
```suggestion
# Why Functional Programming?
```